src/components/icons/controller.icons.ts

```diff
@@ -101,9 +101,45 @@ const getFile = async (req: Express.Request, res: Express.Response, next: Expres
   }
 }
 
+const getFont = async (req: Express.Request, res: Express.Response, next: Express.NextFunction) => {
+  try {
+    const theme = req.query.theme as iconsTheme
```

Comment:
same thing here, i missed the recent PR's i guess. if theme is present everywhere in param, we can have it this way.

Reply:
I will merge it, and fix it with other APIs in a separate PR.